DataFormats/BTauReco/interface/TemplatedSoftLeptonTagInfo.h

```diff
@@ -25,7 +25,8 @@ class SoftLeptonProperties {
         deltaR(   std::numeric_limits<float>::quiet_NaN() ),
         ratio(    std::numeric_limits<float>::quiet_NaN() ),
         ratioRel( std::numeric_limits<float>::quiet_NaN() ),
-        elec_mva( std::numeric_limits<float>::quiet_NaN() )
+        elec_mva( std::numeric_limits<float>::quiet_NaN() ),
+        charge(   std::numeric_limits<int>::quiet_NaN() )
```

Comment:
but then the default of 0 is not a good value for the charge if it's the same for something meaningless (default) and something meaningful (jet without leptons)